framework/test/gtest/TestUtilListenerHelpers.h

```diff
@@ -0,0 +1,99 @@
+/*=============================================================================
```

Comment:
I don't see the removal of this file from the legacy test driver code. It looks like the template classes are only used in StaticBundleTest, which is ported to gtest. Can `framework\test\driver\TestUtilListenerHelpers.h` be deleted?

Reply:
Yes. Deleted.!